### PR TITLE
adding files from muawi_resnets

### PR DESCRIPTION
**Pull request by vivianwhite**
adding conv_modules, og_pytorch_cifar, and resnet scripts from muawi_resnets branch to new refactor folder

### CODE REVIEW

File: refactor/refactor_rainbow.py
Comment (MuawizChaudhary) [reply refactor/refactor_rainbow.py:56]: I like this suggestion Vivian I'll implement it.

File: refactor/conv_modules.py
Comment (obilaniu) [inline refactor/conv_modules.py:42]: [Why not `torch.empty_like()`?](http://pytorch.org/docs/master/generated/torch.empty_like.html?highlight=empty_like#torch.empty_like)

Comment (MuawizChaudhary) [reply refactor/conv_modules.py]: honestly thats better yeah.

**Review by MuawizChaudhary — COMMENTED** [review]
Vivian in the future when you rename a folder `mv refactor FactConv`, git add the old folder name `git add refactor`

File: V1-models/scripts/s_f_sweep/sweep_data.txt
Comment (MuawizChaudhary) [inline V1-models/scripts/s_f_sweep/sweep_data.txt:1]: I don't see why this file is here.

File: V1-models/scripts/s_f_sweep/sweep_data_script.py
Comment (MuawizChaudhary) [inline V1-models/scripts/s_f_sweep/sweep_data_script.py:1]: I don't see why this file is here.

File: V1-models/scripts/s_f_sweep/sweep_data.py
Comment (MuawizChaudhary) [inline V1-models/scripts/s_f_sweep/sweep_data.py:1]: I don't see why this file is here.

File: V1-models/scripts/s_f_sweep/script2
Comment (MuawizChaudhary) [inline V1-models/scripts/s_f_sweep/script2:1]: I don't see why this file is here.

File: V1-models/scripts/s_f_sweep/script1
Comment (MuawizChaudhary) [inline V1-models/scripts/s_f_sweep/script1:1]: I don't see why this file is here.

File: V1-models/scripts/s_f_sweep/run_model.py
Comment (MuawizChaudhary) [inline V1-models/scripts/s_f_sweep/run_model.py:1]: I don't see why this file is here.

File: V1-models/scripts/s_f_sweep/generate_params.py
Comment (MuawizChaudhary) [inline V1-models/scripts/s_f_sweep/generate_params.py:1]: I don't see why this file is here.

**Review by MuawizChaudhary — COMMENTED** [review]
I don't see why some of these files are here.

**Comment by MuawizChaudhary** [comment]
Please delete these unused models [[1](https://github.com/glomerulus-lab/fact-conv/blob/refactor/FactConv/LC_models.py#L7-L51), [2](https://github.com/glomerulus-lab/fact-conv/blob/refactor/FactConv/LC_models.py#L114-L151)]

How many of the models/scripts [here](https://github.com/glomerulus-lab/fact-conv/tree/refactor/V1-models) are relevant to reproducing the results of your project?

File: FactConv/LC_models.py
Comment (MuawizChaudhary) [inline FactConv/LC_models.py:1]: I'd move this folder into `models` dir

**Comment by MuawizChaudhary** [comment]
These scripts seem useless. [[1](https://github.com/glomerulus-lab/fact-conv/blob/refactor/V1-models/BN_V1_V1_ResNet.py), [2](https://github.com/glomerulus-lab/fact-conv/blob/refactor/V1-models/Learnable_Cov_BN_V1_V1_Linear_MNIST.py), [3](https://github.com/glomerulus-lab/fact-conv/blob/refactor/V1-models/Scattering_MNIST.py), [4](https://github.com/glomerulus-lab/fact-conv/blob/refactor/V1-models/Uniform_MNIST.py), [5](https://github.com/glomerulus-lab/fact-conv/blob/refactor/V1-models/V1_MNIST.py), [6](https://github.com/glomerulus-lab/fact-conv/blob/refactor/V1-models/whitening_incremental_V1_MNIST.py) ] 

Can you verify that all the scripts in this screenshot are needed?

**Comment by MuawizChaudhary** [comment]
Please delete this [folder](https://github.com/glomerulus-lab/fact-conv/tree/refactor/scattering-models) given that you moved the one used scattering architecture.

File: README.md
Comment (MuawizChaudhary) [inline README.md:2]: "The important code relevant" could be better phrased. "The most important used in the paper"?

File: FactConv/RSN-scripts/LC_CIFAR100.py
Comment (MuawizChaudhary) [inline FactConv/RSN-scripts/LC_CIFAR100.py:47]: I'm not a fan of writing stuff to your user space rather than your scratch space. if you want this useable by other people, consider adding a argument specifying scratch dir, implement some sort of reading from config , or leave it to the people who use your scripts to hardcode their own scratch dirs.

File: requirements.txt
Comment (MuawizChaudhary) [inline requirements.txt:1]: What is this package for?

File: requirements.txt
Comment (MuawizChaudhary) [inline requirements.txt:6]: Ditto.

File: requirements.txt
Comment (MuawizChaudhary) [inline requirements.txt:9]: Ditto.

**Review by MuawizChaudhary — COMMENTED** [review]
Mind explaining some of these packages?

**Comment by MuawizChaudhary** [comment]
Kinda realziing that the scripts [here](https://github.com/glomerulus-lab/fact-conv/tree/refactor/FactConv/RSN-scripts) should be moved back to the `RSN_experiments` dir, what do you think Vivian?

**Review by MuawizChaudhary — APPROVED** [review]
I'm pro merging at this point, pending a review by @kamdh.

**Comment by MuawizChaudhary** [comment]
Given that we are presenting today at Re-Align, I am squashing and merging. 

@kamdh if you wish to investigate the individual commits, a copy of the commits are located [here](https://github.com/glomerulus-lab/fact-conv/tree/copy_refactor) in `copy_refactor`.

**Pull request merged**
